### PR TITLE
Bugfix/fixing failures

### PR DESCRIPTION
**Pull request by Magnificent-Big-J**
Testing and fixing bugs

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -24,15 +24,5 @@
         "ext-json": "*",
         "php": "^7.4|8.*",
         "friendsofphp/php-cs-fixer": "^3.10"
```
Comment (brycelarge) [inline composer.json]: the extra part is only useful when used by a laravel system, dont think we need to remove laravel support?

File: src/ColliveryApiRequest/ColliveryApiRequest.php [hunk 2]
```diff
@@ -30,11 +26,11 @@ public function getHeaders(): array
         return [
             'Content-Type' => 'application/json',
             'Accept' => 'application/json',
-            'X-App-Name' => 'My Custom App',
-            'X-App-Version' => '0.2.1',
-            'X-App-Host' => '.NET Framework 4.8',
+            'X-App-Name' => $this->config['app_name'],
+            'X-App-Version' => $this->config['app_version'],
+            'X-App-Host' => $this->config['app_host'],
             'X-App-Lang' => 'C#',
```
Comment (WilRyan) [inline src/ColliveryApiRequest/ColliveryApiRequest.php]: App-Lang needs to be PHP but rather from config so version can be set

**Review by WilRyan — APPROVED** [review]

**Review by drallan — APPROVED** [review]

File: composer.json [hunk 3]
```diff
@@ -24,15 +24,5 @@
...
-    },
```
Comment (brycelarge) [inline composer.json:27]: needed for laravel systems

**Review by brycelarge — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (Magnificent-Big-J) [reply composer.json]: > the extra part is only useful when used by a laravel system, dont think we need to remove laravel support?

Remember D4U is built in Laravel

Comment (brycelarge) [reply composer.json]: ya but your on github this is not D4U specific. If you remove it then others using it within laravel will have to tell laravel about the service provider which you might want to add to the read me and if you leave it you have to exclude that extra in D4U composer package. Ill leave that up to you.

Comment (Magnificent-Big-J) [reply composer.json]: [86f1962]

File: composer.json (hunk 3 above)
Comment (Magnificent-Big-J) [reply composer.json:27]: [51e2c17]

File: src/ColliveryApiRequest/ColliveryApiRequest.php (hunk 2 above)
Comment (Magnificent-Big-J) [reply src/ColliveryApiRequest/ColliveryApiRequest.php]: [38651bc]

**Comment by Magnificent-Big-J** [comment]
Bug fixed

**Pull request merged**